src/figuresClasses.ts

```diff
@@ -1,19 +1,69 @@
-export interface Figure {
+type ShapesType = 'triangle' | 'circle' | 'rectangle';
```

Comment:
```suggestion
type Shapes = 'triangle' | 'circle' | 'rectangle';
```

Reply:
custom types in TS are called starting with capital letter - and from TS syntax it's clear that it's a type

Reply:
Please fix

Reply:
@Bogdan-Mykhailov FYI

Reply:
@OleksiiBlezniuk 👍🏼 @kuznetsova-anastasiia sorry